docs/source/classroom/class_use.md

```diff
@@ -10,23 +10,23 @@ The following Frequently Asked Questions section should help answer most of your
 
 ## Is there any training on the cluster for my class?
 
-Yes, we currently provide online training for your class on using the cluster. In the past, we have also given in-person presentations during a class period on the Boston campus. We provide training each semester. We can customize the training to focus on the resources you will be using with them for the class. Email us at <rchelp@northeastern.edu> and provide details about your class and what training you would like us to provide.
+Yes, we currently provide virtual classroom facilitation for your class on using the cluster. We provide training each semester. We can customize the training to focus on the resources you will be using for the class. Email us at <rchelp@northeastern.edu> and provide details about your class and what training you would like us to provide.
 
 ## Do my students have to learn Linux to work with the cluster?
 
 Depending on your class assignments, many students can work with the Open OnDemand web portal, which does not require any knowledge of Linux to use. In cases where you want them to work on the command line, they should have a basic understanding of Linux commands. Please see the question “Is there any training on the cluster for my class” above if you would like us to provide your class with an introductory training course before using the cluster.
 
 ## What software is available to use with my class on the cluster?
 
-Many software packages are available, including popular software apps such as Jupyter Notebook, RStudio, and MATLAB. If you have an account on the cluster, you can see the list of available software by using the `module avail` command. See {ref}`using-module` for more information. Students have access to all the modules on the cluster. They can also use the interactive apps available on Open OnDemand. See {ref}`using-ood` for more information. We can also install additional modules and libraries for your class as needed.
+Many software packages are available, including popular software apps such as Jupyter Notebook, RStudio, and MATLAB. If you have an account on the cluster, you can see the list of available software by using the `module avail` command (run in the command line). See {ref}`using-module` for more information. Students have access to all the modules on the cluster. They can also use the interactive apps available on Open OnDemand. See {ref}`using-ood` for more information. Instructors can install many other software packages using {ref}`conda`.
```

Comment:
We might want to be careful on all modules. There are some modules that are licensed and require you to be parts of certain groups. I am not sure if they are chowned in such a way they won't be visible or not.